content/1_General/Working_MDX.mdx

```diff
@@ -1,68 +1,19 @@
-# Content Documentation
-
-Note: You may find [this live editor](https://usaco.guide/editor) helpful when
-writing modules.
-
-The following two modules are good to read:
-
-1. https://usaco.guide/general/modules
-2. https://usaco.guide/general/code-con
-
-All modules are written in
-[Markdown](https://www.markdownguide.org/cheat-sheet/). There are special
-additions to the markdown that we have added to this site. If you are confused
-about something, or if there's a certain feature that you want to add, reach out
-to Nathan Wang.
-
-## 1. Getting Started
-
-### Using the Editor (recommended)
-
-The easiest way to edit content is to use our live
-[editor](https://usaco.guide/liveupdate). If you choose this option, you can
-skip the rest of the "Getting Started" section.
-
-### Running Site Locally
-
-You can also [run a local version of the site](Front%20End%20Documentation.md)
-in order to view your changes. This is useful if you need to use Tailwind CSS
-classes, which don't work with /editor.
-
-### Using Sublime Text with .mdx
-
-1. Download [here](https://www.sublimetext.com/).
-2. Open the command palette (Cmd-Shift-P) and install package control.
-3. Open the command palette again, install package -> PackageResourceViewer
-4. Extract the HTML package with PackageResourceViewer.
-5. Now you can modify `html_completions.py` by adding to `normal_tags` (ex.
-   `spoiler`)
-   - actually, for some reason uppercase (`CPPSection`) doesn't work ...
-6. Open a `.mdx` file and set syntax highlighting to be the same as `.md` with
-   `View -> Syntax -> Open all with current extension as ... -> Markdown -> Markdown`.
-7. Make snippets!
-
-### Other Tools
-
-- You can use [StackEdit](https://stackedit.io/) to check that latex renders
-  properly.
-- [Markdown Table Generator](https://www.tablesgenerator.com/markdown_tables)
-
-## 2. Module Ordering
-
-### Organization
-
-There are six **sections** in the guide: Intro, Bronze, Silver, Gold, Platinum,
-and Advanced. Each section is subdivided into **categories**, which is just a
-group of modules. Each **module** represents one "article" or "page" on the
-site.
-
-### `ordering.ts`
+---
+id: working-mdx
+title: Working With MDX
+author: Nathan Wang
+description:
+  'Explanation of the frontmatter that precedes every module and solution, a
+  list of custom components that may be used within modules or solutions.'
+prerequisites:
+  - modules
+---
 
-Located at `content/ordering.ts`, this file stores the ordering of the modules.
-Hopefully the format is self-explanatory (it matches based on "slug"). Let
-Nathan Wang know if you have questions.
+We're using [MDX](https://mdxjs.com/), a superset of Markdown. HTML and React
```

Comment:
maybe mention somewhere that the compiler is xdm so `<` and `>` need to be escaped